Automattic/wp-calypso
# Site Settings: Move Posts Per Page field to Custom Content Types

### Purpose

As suggested in #13568, this PR moves the regular **Posts per Page** field to the **Custom Content Types** card and renames that card to **Content Types**. 

### Rationale

Rationale behind this change is simple: all posts per page fields to be managed in the same place, and in a consistent way. A good consequence from that change is that we're able to remove some code, because we're reusing what we can from the current **Custom Content Types** card.

Fixes #14910.

### Preview

**After (the new Custom Content Types card)**
![](https://cldup.com/CEwMGoKHU4.png)

**Before (the old Posts Per Page field in the Composing card)**
![](https://cldup.com/BvYdJR-uLp.png)

**Before (the old Custom Content Types card)**
![](https://cldup.com/o-SIqnmGTq.png)

### To test
* Checkout this branch, or get it going on calypso.live.
* Go to `/settings/writing/:site`, where `:site` is a Jetpack site.
* Verify you see the Posts Per Page field in the new location (**Custom Content Types** card), and not in the old one (**Composing** card).
* Verify the new **Posts Per Page** field saves and retrieves the value correctly.
* Test the above for a WP.com site.
